shared/services/rewards/types.go

```diff
@@ -143,6 +143,7 @@ type IntervalInfo struct {
 	CID                    string        `json:"cid"`
 	StartTime              time.Time     `json:"startTime"`
 	EndTime                time.Time     `json:"endTime"`
+	TotalNodeWeight        *QuotedBigInt `json:"totalNodeWeight"`
```

Comment:
```suggestion
	TotalNodeWeight        *QuotedBigInt `json:"-"`
```

unless you want to update the UnclaimedIntervals and InvalidIntervals code in `shared/types/api/node.go` i would just skip serializing this.

Reply:
There's only 1 commit. Did you push the latest changes?

Reply:
No, not yet. Will likely push next in an hour or so

Reply:
@0xfornax I'm not sure how long it will take to fix this, so don't block the next smartnode release or anything